rhoas-cli-service-registry/README.adoc

```diff
@@ -0,0 +1,371 @@
++
+--
+This example switches to the `my-registry` instance that was previously created.
+
+.Selecting a Service Registry instance to use
+[source,shell]
+----
+$ rhoas service-registry use --name my-registry
+Service Registry instance "my-registry" has been set as the current instance.
+----
+--
+
+. Upload a Service Registry artifact.
++
+--
+This example uploads a Service Registry artifact called `my-artifact` to the Service Registry instance.
+The artifact is an Apache Kafka Avro schema in JSON format.
+
+.Uploading an artifact
+[source,shell]
+----
+wget https://github.com/redhat-developer/app-services-cli/blob/main/docs/resources/avro-userInfo.json
+$ rhoas service-registry artifact create --type=AVRO --artifact-id=my-artifact avro-userInfo.json
+----
+--
+
+. Verify that the artifact was registered.
++
+--
+This example lists all artifacts belonging to the Service Registry instance.
+
+.Listing artifact details for a Service Registry instance
+[source,shell]
+----
+rhoas service-registry artifact list
+Using default artifacts group.
+  ID            NAME   CREATED ON                 CREATED BY   TYPE   STATE
+ ------------- ------ -------------------------- ------------ ------ --------
+  my-artifact          2021-09-27T16:17:35+0000      User      AVRO   ENABLED
+----
+--
+
+. Check the version and metadata information for the artifact.
++
+--
+.Checking the version information of an artifact
+[source,shell]
+----
+rhoas service-registry artifact versions --artifact-id=my-artifact
+{
+  "count": 1,
+  "versions": [
+    {
+      "contentId": 29,
+      "createdBy": "User",
+      "createdOn": "2021-09-27T16:17:35+0000",
+      "globalId": 28,
+      "state": "ENABLED",
+      "type": "AVRO",
+      "version": "1"
+    }
+  ]
+}
+----
+
+.Checking the metadata information of an artifact
+[source,shell]
+----
+rhoas service-registry artifact metadata-get --artifact-id=my-artifact
+Using default artifacts group.
+Fetching artifact metadata
+✔️ Successfully fetched artifact metadata
+{
+  "contentId": 29,
+  "createdBy": "User",
+  "createdOn": "2021-09-27T16:17:35+0000",
+  "globalId": 28,
+  "id": "my-artifact",
+  "modifiedBy": "User",
+  "modifiedOn": "2021-09-27T16:17:35+0000",
+  "state": "ENABLED",
+  "type": "AVRO",
+  "version": "1"
+}
+----
+
+[NOTE]
+====
+You can use additional options, such as `--group`, `--version`, and  `--description`, to modify the metadata of the artifact you're creating.
+For more information about any of the options, view the command help `rhoas service-registry artifact metadata-set -h`.
+====
+--
+
+[id="proc-downloading-service-registry-artifacts_{context}"]
+== Downloading Service Registry artifacts
+
+[role="_abstract"]
+After registering Service Registry artifacts, you can download them to make updates.
+
+.Prerequisites
+
+* You have created a Service Registry instance with at least one artifact.
+
+.Procedure
+
+* Download the artifact by specifying its ID or global ID.
```

Comment:
Shall we explain why we would use one ID over the other? (Not sure)

Reply:
```suggestion
* Download the artifact by specifying Artifact ID that we specified when creating artifact
* Download the artifact by specifying Global ID retrieved from Kafka Message payload
```